aiocache/factory.py

```diff
@@ -1,25 +1,31 @@
-from copy import deepcopy
+import importlib
+import warnings
+from types import MappingProxyType
+from typing import MutableMapping
 
 
 def _class_from_string(class_path):
     class_name = class_path.split('.')[-1]
     module_name = class_path.rstrip(class_name).rstrip(".")
-    return getattr(__import__(module_name, fromlist=[class_name]), class_name)
+    return getattr(importlib.import_module(module_name), class_name)
 
 
 def _create_cache(cache, serializer=None, plugins=None, **kwargs):
-
     if serializer is not None:
-        cls = serializer.pop("class")
+        cls = serializer['class']
         cls = _class_from_string(cls) if isinstance(cls, str) else cls
-        serializer = cls(**serializer)
+        serializer = cls(
+            **{k: v for k, v in serializer.items() if k != 'class'}
+        )
```

Comment:
how is this better than before? Previous version looks simpler

Reply:
I guess we can't use `pop()` anymore since the dict is not copied, and we would be modifying it.

Reply:
Yes, exactly, it was changed to avoid `pop()` usage here